plugins/greatexpectations/flytekitplugins/greatexpectations/schema.py

```diff
@@ -0,0 +1,294 @@
+import datetime
+import logging
+import os
+import typing
+from dataclasses import dataclass
+from typing import Type
+
+import great_expectations as ge
+from dataclasses_json import dataclass_json
+from great_expectations.checkpoint import SimpleCheckpoint
+from great_expectations.core.run_identifier import RunIdentifier
+from great_expectations.core.util import convert_to_json_serializable
+from great_expectations.exceptions import ValidationError
+
+from flytekit import FlyteContext
+from flytekit.extend import TypeEngine, TypeTransformer
+from flytekit.models import types as _type_models
+from flytekit.models.literals import Literal, Primitive, Scalar
+from flytekit.models.types import LiteralType
+from flytekit.types.file.file import FlyteFile, FlyteFilePathTransformer
+from flytekit.types.schema.types import FlyteSchema, FlyteSchemaTransformer, SchemaOpenMode
+
+
+@dataclass_json
+@dataclass
+class BatchConfig(object):
```

Comment:
Could you explain the difference between this and `BatchRequestConfig` in `task.py`?

Reply:
My bad! Removed `BatchConfig` in `schema.py`. Now using `BatchRequestConfig` in both the files.